src/actions/MenuActions.js

```diff
@@ -3,6 +3,9 @@ import axios from 'axios';
 
 // REQUESTS
 
+let breadcrumbsCache = {};
```

Comment:
question (mind that i'm a javascript-noob): how and when is this cache discarded or invalidated?